routes/index.js

```diff
@@ -213,4 +213,84 @@ router.post('/save-journal', async function(req, res, next) {
 
 });
 
+/*
+
+put request expects req.body with following format:
+
+ {"section": 
+        {
+            
+            "journal_reference_id": XX, 
+            "section_number": XX,
+            "contentDetails": {
+                "content": "CONTENT" 
+            }
+        }
+      }
+*/
+
+router.post('/save-section', async function(req, res, next){
+  //initiates connection to database
+  const client = await pool.connect()
+
+  //parses data from request body
```

Comment:
Perhaps there isn't a need to pass in a `sections` property but just the properties directly into the req.body (e.g. instead of `req.body.sections[key]` it would be req.body[key])